server/src/uuid/model/entity/mod.rs

```diff
@@ -1057,6 +1057,73 @@ impl Entity {
     }
 }
 
+impl Entity {
+    pub async fn entity_set_license<'a, E>(
+        payload: &entity_set_license_mutation::Payload,
+        executor: E,
+    ) -> Result<(), operation::Error>
+    where
+        E: Executor<'a>,
+    {
+        let mut transaction = executor.begin().await?;
+
+        sqlx::query!(
+            r#"
+                select * from user where id = ?
+            "#,
+            payload.user_id,
+        )
+        .fetch_optional(&mut transaction)
+        .await?
+        .ok_or(operation::Error::BadRequest {
+            reason: format!("An user with id {} does not exist.", payload.user_id),
+        })?;
```

Comment:
Indeed that is good for [this purpose](https://github.com/serlo/serlo.org-database-layer/issues/224#issuecomment-1124981732) but we should think later in a solution that changes the EventPayload. [here](https://github.com/serlo/serlo.org-database-layer/blob/main/server/src/event/model/event.rs#L162-L176)  we get a sql contraint error, that leads to the internal server error. You could try to catch and convert the error in this PR or we could open a refactor issue for that...

Reply:
Let's do it afterwards. I've opened an issue for that #264